test/e2e/acceptance.browser.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { test, expect } from "@playwright/test";
```

Comment:
Interesting, what does this testing bring us on top of what we have now? We already use `nock` in our unit tests so we are testing that each call actually goes out the library.